content/pt/blue_green_deployment.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Blue Green Deployment
+status: Completed
+category: concept
+---
+
+## O que é
+
+O Blue-green deployment é uma estratégia para atualizar os sistemas de computador em execução com o mínimo de tempo de inatividade. O operador mantém dois ambientes, chamados "blue" e "green".
```

Comment:
What you think about changing the "downtime" translation from "tempo de inatividade" to "tempo de indisponibilidade" ?
"Inatividade" could be also interpreted as no access from clients instead of application down.